PowerPointLabs/PowerPointLabs/ThisAddIn.cs

```diff
@@ -726,18 +726,18 @@ private void UpdateELearningPane(int selectedSlidesCount)
                 return;
             }
             ELearningLabTaskpane taskpane = elearningLabPane.Control as ELearningLabTaskpane;
-            if (selectedSlidesCount > 0)
-            {
+         //   if (selectedSlidesCount > 0)

```

Comment:
Should these commented lines be removed?

Reply:
Thanks! Removed commented lines.